### PR TITLE
feat: optimise structuring and common used dev-utils

### PR DESCRIPTION
**Pull request by neutrinoks**
- `assets.rs` got renamed into `mock_utils.rs`
- commonly used methods and constants for creating test account IDs and addresses are centralised in `mock_utils.rs`
- no usage of `Public` anymore, instead we directly use the trait `Ss58Codec` as trait bound

### CODE REVIEW

File: pallet/src/benchmarking.rs [hunk 1]
```diff
@@ -1,46 +1,40 @@
...
-            multiple_signers_init_module(&bob_mv),
-            car_wash_register_new_user(&alice_mv),
-            car_wash_buy_coin(&alice_mv, 1),
-            car_wash_wash_car(&alice_mv),
-            multiple_signers_rent_apartment(&alice_mv, &dave_mv, &eve_mv, 1),
-            multiple_signers_rent_apartment(&alice_mv, &dave_mv, &eve_mv, 1),
-            multiple_signers_rent_apartment(&alice_mv, &dave_mv, &eve_mv, 1),
+            car_wash_initial_coin_miniting().to_vec(),
+            multiple_signers_init_module().to_vec(),
+            car_wash_register_new_user().to_vec(),
```
Comment (Rqnsom) [inline pallet/src/benchmarking.rs:32]: Where are those users gone? Isn't this supposed to be a transaction script here? 🤔

**Review by Rqnsom — APPROVED** [review]
LGTM

File: pallet/src/benchmarking.rs (hunk 1 above)
Comment (neutrinoks) [reply pallet/src/benchmarking.rs:32]: That is a long story... I switched from bytecode scripts and generating the transaction scripts to load the out-of-the-box transaction scripts. The main reason is that I was trying to move the code in `benchmark_only` also to `mock_utils`, but this seems not to be possible 😩

**Review by Rqnsom — APPROVED** [review]
LGTM

**Pull request merged**